exonum-java-binding/tests_profile.py

```diff
@@ -0,0 +1,111 @@
+import sys
+import os
+import fnmatch
+import subprocess
+
+
+class TestsProfile:
+    def __init__(self):
+        self.shared_lib_extension = self._shared_lib_extension()
+        self.jvm_lib_name = self._jvm_lib_name()
+        self.java_home = self._get_java_home()
+        self.java_lib_dir = self._get_java_lib_dir()
+        self.rust_sysroot = self._get_rust_sysroot()
+        self.std_lib_dir, self.std_lib_name = self._get_std_lib_location()
+        self.project_root = self._get_project_root()
+
+        self.java_binding_lib_name = self._java_bindings_lib_name()
+        self.exonum_java_name = self._exonum_java_name()
+
+        self._set_environment_variables()
+
+    def _set_environment_variables(self):
+        os.environ["JAVA_HOME"] = self.java_home
+        print(f"JAVA_HOME={self.java_home}")
+        self._set_rustflags()
+        os.environ["PATH"] = self._create_path()
```

Comment:
We didn't use to modify `PATH` (on Linux/Mac). Why is that needed?

Reply:
It's needed on Windows, but yes, it must be removed on Linux/Mac